.github/pull_request_template.md

```diff
@@ -8,9 +8,24 @@ Fixes #
 
 <!-- Explain the changes you made, and why they're needed. Add a screenshot if you've made any UI changes!  -->
 
-# 🧪 How has/should this change been tested?
+# 🧪 Testing
 
-<!-- Explain how to setup for testing here if it is not already obvious, and how you've tested this PR. -->
+<!-- Explain how to setup for testing here if it is not already obvious, and steps you've taken to test this PR. -->
+
+## Setup
+
+<!-- Explain how to setup for testing here if it is not already obvious. -->
+
+- [ ] _(e.g.)_ Central Sync server with 1 Legacy Desktop remote site and 1 OMS remote site running this PR
+- [ ] _(e.g.)_ This sample datafile:
```

Comment:
would be good to encourage sharing of datafiles and/or sqlite database or pg backup via google drive. repro of bugs is always an issue!